python/ngen_cal/tests/conftest.py

```diff
@@ -219,3 +222,32 @@ def explicit_catchments(nexus, fabric, workdir) -> Generator[ List[ CalibrationC
         cat = CalibrationCatchment(workdir, id, nexus, start, end, fabric, 'Q_Out', eval_options, data)
         catchments.append(cat)
     yield catchments
+
+@pytest.fixture
+def multi_model_shared_params() -> Mapping[str, list[Parameter]]:
+    p1 = Parameter(name='a', min=0, max=1, init=0)
+    p2 = Parameter(name='d', min=2, max=3, init=0)
+    p3 = Parameter(name='c', min=0, max=1, init=0)
+    p4 = Parameter(name='a', min=0, max=1, init=0)
+    params = {'A':[p1], 'B':[p2, p4], 'C':[p3, p4]}
+
+    return params
+
+@pytest.fixture
+def multi_model_shared_params2() -> Mapping[str, list[Parameter]]:
```

Comment:
At first glance, this overlaps with `multi_model_shared_params` s.t. I dont think we need it? Can we just get rid of this fixture?